File: 250408/두 개의 격자 비교하기/compare-two-grid.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int m = sc.nextInt();
        int arr1[][] = new int[10][10];
        int arr2[][] = new int[10][10];
        for(int i = 0; i < n; i++) {
            for(int j = 0; j < m; j++) {
                arr1[i][j] = sc.nextInt();
            }
        }
        for(int i = 0; i < n; i++) {
            for(int j = 0; j < m; j++) {
                arr2[i][j] = sc.nextInt();
            }
        }
        int answer[][] = new int[10][10];
        for(int i = 0; i < n; i++) {
            for(int j = 0; j < m; j++) {
                if(arr1[i][j] == arr2[i][j]) {
                    answer[i][j] = 0;
                } else {
                    answer[i][j] = 1;
                }
                System.out.print(answer[i][j] + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}

File: 250408/두 개의 격자 비교하기/compare-two-grid.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int n = 0, m = 0;
    int arr1[10][10] = {};
    int arr2[10][10] = {};
    int answer[10][10] = {};
    cin >> n >> m;
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < m; j++) {
            cin >> arr1[i][j];
        }
    }
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < m; j++) {
            cin >> arr2[i][j];
        }
    }
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < m; j++) {
            if(arr1[i][j] == arr2[i][j]) {
                answer[i][j] = 0;
            } else {
                answer[i][j] = 1;
            }
            cout << answer[i][j] << " ";
        }
        cout << "\n";
    }
    return 0;
}